### PR TITLE
Updated download page for 4.0 release.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Review by carltongibson — APPROVED** [review]
Good. Thanks. Exciting 💃

**Comment by felixxm** [comment]
Thanks :+1:

File: djangoproject/templates/releases/download.html [hunk 1]
```diff
@@ -122,6 +116,12 @@ <h2 id="future-versions">Future Roadmap</h2>
       <td>December 2023</td>
       <td>April 2026</td>
     </tr>
+    <tr>
+      <td>5.0 LTS</td>
```
Comment (timgraham) [inline djangoproject/templates/releases/download.html:120]: chop "LTS"

Comment (felixxm) [reply djangoproject/templates/releases/download.html:120]: Good catch, thanks :dart:

**Pull request merged**